src/mesh/config/MeshConfiguration.cpp

```diff
@@ -109,7 +110,7 @@ void MeshConfiguration::addMesh(
         break;
       }
     }
-    PRECICE_CHECK(found, "Data " << dataNewMesh->getName() << " is not available in data configuration!");
+    PRECICE_CHECK(found, "Data " << dataNewMesh->getName() << " is not defined. Please define a data tag with name=\""<< dataNewMesh->getName() << "\".");
```

Comment:
Seems like this error won't be run into, but good to keep it? I always hit the error above first.

Reply:
I also don't understand why this check is here, but I also don't see any problem with keeping it. Let's keep it then?

Reply:
@KyleDavisSA @uekerman shall we keep this then?

Reply:
Seems like this function is only called in tests. We could convert it to an assertion.

Reply:
I converted this to an assertion, thanks!